test_util/src/builders.rs

```diff
@@ -409,41 +418,207 @@ impl TestCommandBuilder {
 
   #[track_caller]
   pub fn run(&self) -> TestCommandOutput {
```

Comment:
I pushed a lot of the functionality in here down into `DenoCmd`. It would be good to eventually unify the APIs somehow, but for now this is at least better.

(`DenoCmd` is the old API, but sometimes it's nice when to use when you just want to run a command in a test, so this helps out with that)